waspc/test/AppSpec/EntityTest.hs

```diff
@@ -8,10 +8,17 @@ import qualified Wasp.Psl.Ast.Model as PslModel
 spec_AppSpecEntityTest :: Spec
 spec_AppSpecEntityTest = do
   describe "getIdField" $ do
-    it "gets primary field from entity when it exists" $ do
```

Comment:
We have 19 test files where we write the first letter as upper case and 21 files where we write it as lower case.

Obviously we have two competing standards 😄 @sodic @Martinsos

Reply:
Ha I also wasn't sure, but sounds good it if is nicer to read! Git messages yes, for sure, upper letter and dot at the end!